content/docs/user-guide/networking/serializers.md

```diff
@@ -0,0 +1,75 @@
+---
+title: Network Serializers
+linktitle: Serializers
+weight: 300
+description: An overview and reference for data serializers used by the Open 3D Engine networking stack.
+---
+
+Open 3D Engine supports a variety of Serializers for visiting an object hierarchy and performing operations upon that hierarchy, typically reading from or writing data to the object hierarchy for reasons of persistence or network transmission.
```

Comment:
In this context, "Serializers" (and "Serialization" in the next paragraph) refer to a concept, not a specific class name or tool, therefore capitalization is not needed.